build.js

```diff
@@ -664,7 +670,11 @@ const buildIDLMemberTests = (
       }
     }
 
-    tests[member.name] = compileTest({
+    const name = isEventHandler ?
+      `${member.name.replace('on', '')}_event` :
```

Comment:
Here, can we instead check `member.name.startsWith('on')` and then let `name = member.name.subtring(2)`?

Or we could include member.name.startsWith('on') in the definition of `isEventHandler`, so that we don't have to deal with the hypothetical case where it's not true.

Reply:
I tweaked the `replace()` call to use a small bit of regex instead that checks for `on` in the beginning of the string, which should do the above while still keeping the code to one line!